templates/blog/post_list.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <h2>All post:</h2>
```

Comment:
The <h2> tag should be inside the <body> tag, not the <head> tag. Move this line to the appropriate section.